index.bs

```diff
@@ -420,13 +425,22 @@ for the name and value of the header are described by the following ABNF
 grammar [[!RFC5234]]:
 
 <pre dfn-type="grammar" link-type="grammar">
-    suborigin = 1*( <a>LOWERALPHA</a> / <a>DIGIT</a> / "-" )
+    <dfn>suborigin-name</dfn> = 1*( <a>LOWERALPHA</a> / <a>DIGIT</a> / "-" )
+    <dfn>suborigin-policy-option</dfn> = "'unsafe-postmessage-send'"
+                              / "'unsafe-postmessage-receive'"
```

Comment:
well, in general we can add it in later anyhow. My concern is that i am not sure in how many cases one is fine and not the other. If it is, that probably means someone somewhere is not checking the origin or not setting the origin correctly.